python_ta/reporters/templates/template.html

```diff
@@ -14,11 +14,13 @@ <h1 class="title">PyTA Error Report</h1>
                 <time>{{ date_time }}</time>
             </header>
             <section class="sidebar">
-                <h2>Table of Contents</h2>
+                <article class="section-header">
```

Comment:
It's definitely good to sync this up with the file reports on the left-hand side. Looking at this, `<article>` isn't the right HTML tag to use here. I think you can just put the `section-header` class on the `h2` tag itself, both here and below, or if that doesn't work I guess you can use a `div`.